test/js/reducers/questionnaire.spec.js

```diff
@@ -646,7 +642,7 @@ describe('questionnaire reducer', () => {
 
     const languageSelection = resultState.data.steps[0]
     expect(languageSelection.type).toEqual('language-selection')
-    expect(languageSelection.choices).toInclude('fr')
+    expect(languageSelection.languageChoices).toInclude('fr')
```

Comment:
I was forced to rename `choices` to `languageChoices` for language selection steps because of Flow. I wanted Step to be union of MultipleChoiceStep and LanguageSelectionStep and that (AFAICT) will only let you tag the union based on values (eg: type === 'language-selection' vs. type === 'multiple-choice'), having two types in the union with a property with the same name and different type apparently messes up the type system.

Multiple Choice Steps have complex `choices`, with values that change depending on mode and language, whereas Language Selection Steps have string `choices`. I don't know if that was on purpose, but in order to merge everything in a sane way I had to properly type choices in both cases, which ultimately led me to rename `choices` in Language Selection Steps to `languageChoices`.